bindings/rust/s2n-tls-tokio/src/lib.rs

```diff
@@ -233,6 +233,45 @@ where
     }
 }
 
+impl<S, C> TlsStream<S, C>
+where
+    C: AsRef<Connection> + AsMut<Connection> + Unpin,
+    S: AsyncRead + AsyncWrite + Unpin,
+{
```

Comment:
nit: why is this in a separate impl block? it has the same constraints as the previous.